src/create_character/create_character.cpp

```diff
@@ -107,11 +109,19 @@ void CreateCharacter(MainCharacterSharedPtr main_character)
 	utils::GetUserInput(user_answer);
 	main_character->set_name(user_answer);
 
-	utils::Print({"Please give me your race: "});
+	utils::Print({"Please choose your race from the available options: "});
+	for (const auto player_race : main_character_race::AVAILABLE_MCRACES_LIST)
+	{
+		utils::Print({main_character_race::MCRaceToString(player_race)});
+	}
```

Comment:
The game don't stop me if i don't type the correct race, it just set's it to undefined and proceeds